phys2denoise/cli/run.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Parser for phys2denoise."""
+
+
+import argparse
+
+from phys2denoise import __version__
+
+
+def _get_parser():
+    """
+    Parse command line inputs for this function.
+
+    Returns
+    -------
+    parser.parse_args() : argparse dict
+
+    Notes
+    -----
+    # Argument parser follow template provided by RalphyZ.
+    # https://stackoverflow.com/a/43456577
+    """
+    parser = argparse.ArgumentParser()
+    optional = parser._action_groups.pop()
+    metric = parser._action_groups.pop()
+    required = parser.add_argument_group('Required Argument:')
+    required.add_argument('-in', '--input-file',
+                          dest='filename',
+                          type=str,
+                          help='Path/name of the file containing physiological '
+                               'data, with or without extension.',
+                          required=True)
+    metric.add_argument('-crf', '--crf',
+                        dest='metric_list',
+                        action='append_const',
+                        const='crf',
+                        help='Cardiac response function. Needs the following '
+                             'inputs:sr, os, tl, onset and tr.',
```

Comment:
I know it becomes a long list, but I would use the long version of the inputs...